### PR TITLE
Context Menu Search

### PR DESCRIPTION
**Pull request by codefruitio**
Added automatic search to context menus for movie grid item, series grid item, and tv episodes

### CODE REVIEW

File: Ruddarr/Views/Movies/MovieContextMenu.swift [hunk 1]
```diff
@@ -11,13 +13,31 @@ struct MovieContextMenu: View {
         if let callsheetUrl = callsheet {
             link(name: "Callsheet", url: callsheetUrl)
         }
+        
+        Divider()
+        
+        Button("Search", systemImage: "magnifyingglass") {
```
Comment (tillkruss) [inline Ruddarr/Views/Movies/MovieContextMenu.swift:19]: I think these should match exactly what the buttons say, "Automatic Search" I think.

File: Ruddarr/Views/Series/EpisodeContextMenu.swift [hunk 2]
```diff
@@ -1,21 +1,42 @@
...
     var body: some View {
         link(name: "Trakt", url: traktUrl)
 
         if encodedTitle != nil {
             link(name: "IMDb", url: imdbUrl)
         }
+        
+        Divider()
+        
+        Button("Search", systemImage: "magnifyingglass") {
```
Comment (tillkruss) [inline Ruddarr/Views/Series/EpisodeContextMenu.swift:17]: Same here.

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: Ruddarr/Views/Series/SeriesContextMenu.swift [hunk 3]
```diff
@@ -1,19 +1,42 @@
...
         link(name: "Trakt", url: traktUrl)
         link(name: "IMDb", url: imdbUrl)
         link(name: "TVDB", url: tvdbUrl)
+        
+        Divider()
+        
+        Button("Search Monitored", systemImage: "magnifyingglass") {
+            Task { await dispatchSearch() }
+        }
+        .disabled(!series.monitored)
```
Comment (tillkruss) [inline Ruddarr/Views/Series/SeriesContextMenu.swift:18]: Maybe we should hide the button if it's not monitored?

**Review by tillkruss — CHANGES_REQUESTED** [review]

File: Ruddarr/Views/Series/SeriesContextMenu.swift (hunk 3 above)
Comment (codefruitio) [reply Ruddarr/Views/Series/SeriesContextMenu.swift:18]: Solid point. Happy to make that change this evening

**Pull request merged**